src/main/java/org/opensearch/flowframework/common/CommonValue.java

```diff
@@ -64,6 +64,8 @@ private CommonValue() {}
     public static final String VALIDATION = "validation";
     /** The field name for provision workflow within a use case template*/
     public static final String PROVISION_WORKFLOW = "provision";
+    /** The field name for workflow steps. This field represents the name of the workflow steps to be fetched. */
+    public static final String WORKFLOW_STEP = "workflow_step";
```

Comment:
nit: thoughts on having this `type` or `name`, not a strong suggestion at all, just thinking cause the param is same as the path kinda `_flow_framework/workflow/_steps`

Reply:
No strong opinion here. @dbwiddis, @joshpalis  thoughts?

Reply:
Looking at the docs we kind of make them synonymous.   "Step" seems like an "instance" whereas "Type" seems like a "class".

I don't think we should change this constant, the underlying question is what to put from an API perspective.

Technically we might want to use "type" or "types" as the filtering param, but since the overall list is called "_steps" then an individual step as the param may be better.  Or maybe "step_types"? 

No strong opinion.

Reply:
@amitgalitz let's get the PR in if no concern here. If there are more inputs on the ask, I'll raise a separate PR. Thanks